test/org/loklak/harvester/TwitterScraperTest.java

```diff
@@ -107,16 +107,20 @@ public void testSearch() {
         // Test tweets data with TwitterTweet object
         TwitterScraper.TwitterTweet tweet = (TwitterScraper.TwitterTweet) ftweet_list.iterator().next();
 
-        assertTrue(String.valueOf(tweet.user).contains(tweet_check.get("user")));
-        assertTrue(String.valueOf(tweet.status_id_url).equals(tweet_check.get("status_id_url")));
+        assertTrue(String.valueOf(tweet.getUser()).contains(tweet_check.get("user")));
+        assertTrue(String.valueOf(tweet.getStatusIdUrl()).equals(tweet_check.get("status_id_url")));
 
-        String created_date = dateFormatChange(String.valueOf(tweet.created_at));
+        String created_date = dateFormatChange(String.valueOf(tweet.getCreatedAt()));
         assertTrue(created_date.equals(tweet_check.get("created_at")));
         
-        assertTrue(String.valueOf(tweet.screen_name).equals(tweet_check.get("screen_name")));
-        assertTrue(String.valueOf(tweet.source_type).equals(tweet_check.get("source_type")));
-        assertTrue(String.valueOf(tweet.provider_type).equals(tweet_check.get("provider_type")));
-        assertTrue(String.valueOf(tweet.text).equals(tweet_check.get("text")));
+        assertTrue(String.valueOf(tweet.getScreenName()).equals(tweet_check.get("screen_name")));
+        assertTrue(String.valueOf(tweet.getSourceType()).equals(tweet_check.get("source_type")));
+        assertTrue(String.valueOf(tweet.getProviderType()).equals(tweet_check.get("provider_type")));
```

Comment:
Please use `assertEquals` when you are checking equality of two objects, it looks more intuitive. And `assertEquals` uses `equals` method internally.